test/distributions/robust/arithmetic/performance/multilocale/rvfSlices/helper.notest

```diff
@@ -0,0 +1 @@
+This isn't designed to be tested
```

Comment:
Was there a helper.chpl file that should be committed alongside this?  Or should it be renamed reduceSlicePrivate.notest (since that .chpl file doesn't seem to have a .good file or a .notest)?

Reply:
reduceSlicePrivate.chpl is in a subdirectory `helper/`, so this .notest skips that directory.

Reply:
D'oh!